includes/KSES.php

```diff
@@ -518,6 +518,12 @@ public function filter_kses_allowed_html( $allowed_tags ) {
 				'href'  => true,
 				'theme' => true,
 			],
+			'amp-story-page-outline'    => [
```

Comment:
Somehow missed this typo, which caused this bug: #8784

Reply:
Oops!